api/XPathResult.json

```diff
@@ -76,10 +76,10 @@
               "version_added": true
             },
             "safari": {
-              "version_added": null
+              "version_added": "7"
```

Comment:
`invalidIteratorState` and `resultType` were in https://trac.webkit.org/browser/webkit/trunk/WebCore/xpath/XPathResult.idl?rev=14234 15 years ago, so I think this a problem of attributes on prototypes. We'll need to test an instance here.